liquibase-core/src/main/java/liquibase/command/core/HistoryCommandStep.java

```diff
@@ -37,6 +38,10 @@ public class HistoryCommandStep extends AbstractCliWrapperCommandStep {
                 .build();
         CHANGELOG_FILE_ARG = builder.argument(CommonArgumentNames.CHANGELOG_FILE, String.class)
                 .description("The root changelog").build();
+        FORMAT_ARG = builder.argument("format", HistoryFormat.class)
+                .description("History output format")
```

Comment:
Hi made an update here removing the required property, as this argument is optional. We don't need to force users to do it. If you wish you can just run `liquibase history` and that should be fine too.